src/App.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
+ import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Home from './pages/Home';
+import About from './pages/About';
+import Quiz from './pages/Quiz';
+import Profile from './pages/Profile';
+import Error404 from './pages/Error/Error404';
 
 function App() {
   return (
-    <div>
-      <h1 className="text-2xl bg-lightblue">The Movie Bubble</h1>
-    </div>
+    <>
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route path="/about">
+            <About />
+          </Route>
+          <Route path="/profile">
+            <Profile />
+          </Route>
+          <Route path="/quiz">
+            <Quiz />
+          </Route>
+          <Route path="*" exact={true} component={Error404} />
```

Comment:
this doesn't need to be wildcard *. you can remove the `path` prop and the `exact` prop, putting it at the end will match everything if not matched above